dataprofiler/labelers/column_name_model.py

```diff
@@ -217,17 +241,30 @@ def predict(
             include_label=self._parameters["include_label"],
         )
 
-        if show_confidences:
-            raise NotImplementedError(
-                """`show_confidences` parameter is disabled
-                for Proof of Concept implementation. Confidence
-                values are enabled by default."""
-            )
+        predictions = np.array([])
+        confidences = np.array([])
+
+        # `data` at this point is either filtered or not filtered
+        # list of column names on which we are predicting
+        for iter_value, value in enumerate(data):
+
+            if output[iter_value][0] > self._parameters["positive_threshold_config"]:
+                predictions = np.append(
+                    predictions,
+                    self._parameters["true_positive_dict"][output[iter_value][1]][
+                        "label"
+                    ],
+                )
+
+                if show_confidences:
+                    confidences = np.append(confidences, output[iter_value][0])
```

Comment:
returning in  the same way as other models `dict(preds=np.array(), confs=np.array())`